File: Examples/VisualBasic/ManageBarcodeImages/SetForeColorText.vb
'
' This project uses Automatic Package Restore feature of NuGet to resolve Aspose.BarCode for .NET API reference 
' When the project is build. Please check http://docs.nuget.org/consume/nuget-faq for more information. 
' If you do not wish to use NuGet, you can manually download Aspose.BarCode for .NET API from http://www.aspose.com/downloads, 
' Install it and then add its reference to this project. For any issues, questions or suggestions 
' Please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
'

Namespace Aspose.BarCode.Examples.VisualBasic.ManageBarCodes
    Class SetForeColorText
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_ManageBarCodesImages()
            Dim dst As String = dataDir & Convert.ToString("barcode-SetForeColorText_out.jpg")

            ' Instantiate barcode object
            Dim barCodeBuilder As New BarCodeBuilder()

            ' Set the Code text and text fore color to red for the barcode 
            barCodeBuilder.CodeText = "1234567"
            barCodeBuilder.CodeTextColor = System.Drawing.Color.Red
            Console.WriteLine(Convert.ToString(Environment.NewLine + "Barcode saved at ") & dst)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/ManageBarcodeImages/SetForeColorText.cs
using System.IO;
using System;
using Aspose.BarCode;

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.BarCode for .NET API reference 
when the project is build. Please check https:// ocs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.BarCode for .NET API from http:// ww.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:// ww.aspose.com/community/forums/default.aspx
*/

namespace Aspose.BarCode.Examples.CSharp.ManageBarCodes
{
    class SetForeColorText
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_ManageBarCodesImages();
            string dst = dataDir + "barcode-SetForeColorText_out.jpg";

            // Instantiate barcode object
            BarCodeBuilder barCodeBuilder = new BarCodeBuilder();

            // Set the Code text for the barcode
            barCodeBuilder.CodeText = "1234567";

            // Set the code text fore color to red
            barCodeBuilder.CodeTextColor = System.Drawing.Color.Red;
            Console.WriteLine(Environment.NewLine + "Barcode saved at " + dst);
        }
    }
}

